source/agora/consensus/PreImage.d

```diff
@@ -523,46 +523,51 @@ public void getCycleSeed (in KeyPair pair, in uint validator_cycle, out Hash see
     import agora.utils.WellKnownKeys;
     import std.format;
 
+    // If the validator_cycle is 1008, call special function for that
+    if (validator_cycle == 1008)
+        return getCycleSeed1008(pair, validator_cycle, seed, seed_height);
```

Comment:
Do we still use 1008 ?

Reply:
Just in some unit tests of `agora.node.Ledger`, we use it.